core/src/main/java/com/google/common/truth/ComparableSubject.java

```diff
@@ -27,7 +27,7 @@
 public abstract class ComparableSubject<T extends Comparable> extends Subject {
   /**
    * Constructor for use by subclasses. If you want to create an instance of this class itself, call
-   * {@link Subject#check(String, Object..) check(...)}{@code .that(actual)}.
```

Comment:
What's wrong with what is here? This should render as:

> Constructor for use by subclasses. If you want to create an instance of this class itself, call `check(...).that(actual)`.

Reply:
I think the problem is that we have 2 dots instead of 3. But I think you're right that we want the rendering here, so all this change should do is add the third dot.

Reply:
That already got fixed in a different commit.

Reply:
Ah, ok. Looks like he wanted both, but I agree that we don't want to change the rendering, so if the link is fixed, we're good.